pkg/storage/storage.go

```diff
@@ -2,13 +2,11 @@ package storage
 
 import (
 	"fmt"
-	"path"
-	"strings"
-
 	"github.com/weaveworks/ignite/pkg/apis/ignite/scheme"
 	meta "github.com/weaveworks/ignite/pkg/apis/meta/v1alpha1"
 	"github.com/weaveworks/ignite/pkg/constants"
 	"github.com/weaveworks/ignite/pkg/storage/serializer"
+	"path"
```

Comment:
move this to the group with the other stdlib packages

Reply:
Moved in 650dad1.